R/utils.R

```diff
@@ -1,8 +1,17 @@
-
+# If left is null, use right argument
 `%||%` <- function(lhs, rhs) {
   if (!is.null(lhs)) {
     lhs
   } else {
     rhs
   }
 }
+
+# If left is not null, use 2nd argument, else use last argument
+ifnotnull <- function(inputToCheck, outputIfNotNull, outputIfNull = NULL) {

```

Comment:
It it possible to ensure that the values are only EVALUATED if required.
Or is this the case anyways
e.g.
if I do 

ifnotnull(NULL, stop("ERROR"), print("HELLO"), what happens?

I guess I am going to try ah ah